services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/docker_utils.py

```diff
@@ -172,19 +178,22 @@ async def _parse_and_publish_logs(
     *,
     task_publishers: TaskPublisher,
     progress_regexp: re.Pattern[str],
+    container_processing_progress_weight: float,
```

Comment:
since you do not do any check inside of the function, perhaps the assert above should be added here? i.e. you assume that this value "is meaninful" at this interface.

Reply:
good idea. will add an assertion. I will probably refactor this later to add stages. this is merely a "quick" fix.